packages/mobile/src/home/HomeTokenBalance.test.tsx

```diff
@@ -162,4 +162,44 @@ describe('HomeTokenBalance', () => {
       ]
     `)
   })
+
+  it('renders correctly when fetching the local currency failed', async () => {
+    const store = createMockStore({
+      ...defaultStore,
+      localCurrency: {
+        error: true,
+        exchangeRates: {
+          [Currency.Dollar]: null,
+        },
+      },
+    })
+
+    const tree = render(
+      <Provider store={store}>
+        <HomeTokenBalance />
+      </Provider>
+    )
+
+    expect(tree.queryByTestId('ViewBalances')).toBeFalsy()
+    expect(getElementText(tree.getByTestId('TotalTokenBalance'))).toEqual('₱-')
+
+    expect(store.getActions().length).toEqual(1)
```

Comment:
nit: Is this expectation needed if you are testing the whole array in the next line?

Reply:
Probably not